doc/source/io.rst

```diff
@@ -3965,72 +3982,184 @@ function.
-table, the complete error response from BigQuery is returned which
-can be quite long given it provides a status for each row. You may want
-to start with smaller chunks to test that the size and types of your
-dataframe match your destination table to make debugging simpler.
+                             col_order=['col1', 'col2', 'col3'], projectid)
+
+.. note::
+
+   You can find your project id in the `BigQuery management console <https://code.google.com/apis/console/b/0/?noredirect>`__.
+
+
+.. note::
+
+   You can toggle the verbose output via the ``verbose`` flag which defaults to ``True``.
+
+Writing DataFrames
+''''''''''''''''''
+
+Assume we want to write a DataFrame ``df`` into a BigQuery table using :func:`~pandas.DataFrame.to_gbq`.
+
+.. ipython:: python
+
+   df = pd.DataFrame({'my_string': list('abc'),
+                      'my_int64': list(range(1, 4)),
+                      'my_float64': np.arange(4.0, 7.0),
+                      'my_bool1': [True, False, True],
+                      'my_bool2': [False, True, False],
+                      'my_dates': pd.date_range('now', periods=3)})
+
+  In [4]: df
+  Out[4]: my_bool1	my_bool2	my_dates	my_float64	my_int64	my_string
+       0	True	False	2015-09-10 03:03:45.365258	4	1	a
+       1	False	True	2015-09-11 03:03:45.365258	5	2	b
+       2	True	False	2015-09-12 03:03:45.365258	6	3	c
+
+  In [5]: df.dtypes
+  Out[5]: my_bool1                bool
+          my_bool2                bool
+          my_dates      datetime64[ns]
+          my_float64           float64
+          my_int64               int64
+          my_string             object
+          dtype: object
 
 .. code-block:: python
 
-   df = pandas.DataFrame({'string_col_name' : ['hello'],
-         'integer_col_name' : [1],
-         'boolean_col_name' : [True]})
-   df.to_gbq('my_dataset.my_table', project_id = projectid)
+   df.to_gbq('my_dataset.my_table', projectid)
 
-The BigQuery SQL query language has some oddities, see `here <https://developers.google.com/bigquery/query-reference>`__
+.. note::
 
-While BigQuery uses SQL-like syntax, it has some important differences
-from traditional databases both in functionality, API limitations (size and
-quantity of queries or uploads), and how Google charges for use of the service.
-You should refer to Google documentation often as the service seems to
-be changing and evolving. BiqQuery is best for analyzing large sets of
-data quickly, but it is not a direct replacement for a transactional database.
+   If the destination table does not exist, a new table will be created. The
+   destination dataset id must already exist in order for a new table to be created.
 
-You can access the management console to determine project id's by:
-<https://code.google.com/apis/console/b/0/?noredirect>
+The ``if_exists`` argument can be used to dictate whether to ``fail``, ``replace``
+or ``append`` if the destination table already exists. The default value is ``fail``.
```

Comment:
Can you quote the fail, replace, append? (so ```` ``'fail'`` ````, then it is more clear it is a value and not a keyword)